modules/HardwareDrivers/PowerSupplies/Huawei_V100R023C10/Huawei_V100R023C10.hpp

```diff
@@ -99,6 +104,8 @@ class Huawei_V100R023C10 : public Everest::ModuleBase {
     };
     // PSU upstream voltage source
     UpstreamVoltageSource upstream_voltage_source;
+
+    std::shared_ptr<fusion_charger::telemetry::TelemetryPublisherBase> telemetry_publisher;
```

Comment:
To me it looks like this doens't need to be a member of this class but can instead be a unique_ptr in `DispenserConfig`